### ISSUE TITLE
Make new error messages in Clojure 1.10 clickable

### ISSUE THREAD

**Issue by cursive-ide** [cursive-ide/cursive#2142]
In Clojure 1.10 the error messages got a significant improvement. However they no longer look like stacktrace lines, so Cursive should also match those to make the source links clickable.

Example:

```
(/ 1 0)
Syntax error (ArithmeticException) compiling at (notification.clj:58:1).
Divide by zero
```

**Comment by cursive-ide** [comment]
This is fixed for the next build. Unfortunately these error messages only have the file name, so for errors coming from e.g. `core.clj` the user will have to select from a dropdown when clicking on the link. It may be possible to address this with later improvements to the REPL infrastructure.

**Comment by puredanger** [comment]
FYI, in 1.10.1 they will have the path too. And of course, the exception sitting in *e has ex-data that includes more attributes as data.

**Comment by cursive-ide** [comment]
@puredanger Yes, I actually realised that after writing that comment, and fixed this appropriately. So in 1.10.1 files should hopefully not be ambiguous and navigation will just work.

I saw that `*e` has more info, but it's tricky to go out asynchronously for it when rendering a hyperlink. I'd really need to store `*e` per evaluation (and that relies on an evaluation having an ID of some kind), and then store the evaluation ID in the hyperlink so I could get the correct exception back later, or something. Unfortunately storing the information temporarily on the server is fundamentally racy, so I'm very pleased to see the path in 1.10.1.
